### ISSUE TITLE
License Plate is not detected or No box is drawn after custom training for License plate

### ISSUE THREAD

**Issue by choudharyvikram** [pythonlessons/TensorFlow-2.x-YOLOv3#38]
I have trained a model for detecting license plate. I have used 69 images for training and 25 for testing 
parameter used : 
TRAIN_BATCH_SIZE = 10
TRAIN_LR_INIT = 9e-5
TRAIN_LR_END = 9e-7
TRAIN_EPOCHS = 10

the last lines of training are : 
giou_val_loss:   1.83, conf_val_loss: 215.04, prob_val_loss:   1.10, total_val_loss: 217.97

calculating mAP45...

0.000% = license-plate AP  
mAP = 0.000%, 6.86 FPS.

Where am i going wrong?

**Comment by pythonlessons** [comment]
Did you use TRAIN_TRANSFER as True from configs.py?
You started from quite a low learning rate, you should train it for longer with that learning rate. Also, you ended up with  total_val_loss: 217.97, it must be lower than 40 at least. Get more training dataset.

**Comment by choudharyvikram** [comment]
TRAIN_TRANSFER was False.
I started with a low LR because when the LR was 1e-4 i was getting total_val_loss as 'nan', so I reduced it.

**Comment by pythonlessons** [comment]
Try to use TRAIN_TRANSFER as True

**Comment by choudharyvikram** [comment]
It worked after changing TRAIN_TRANSFER. Thank you.
What does TRAIN_TRANSFER do exactly?

**Comment by pythonlessons** [comment]
You use transfer_learning (using orinigal YOLO weigts and training model on them) otherwise you training the whole model from zero. Nice that it worked!
